spec/requests/reactions_spec.rb

```diff
@@ -190,17 +190,33 @@
     context "when reacting to an article" do
       before do
         sign_in user
-        post "/reactions", params: article_params
```

Comment:
I removed this and updated the existing specs because `change` assertions are generally a bit more robust than asserting on specific counts. I also dislike specs that are only assertions without any actual actions in them because they are hidden away in `before` blocks.